tests/test_codebuild/test_codebuild.py

```diff
@@ -0,0 +1,704 @@
+        artifacts=artifacts,
+        environment=environment,
+        serviceRole=service_role,
+    )
+    response = client.start_build(
+        projectName=name,
+        sourceVersion=branch_override,
+        artifactsOverride=artifacts_override,
+    )
+
+    response.should_not.be.none
+    response["build"].should_not.be.none
+    response["build"]["sourceVersion"].should.equal("fix/testing")
+
+
+@mock_codebuild
+def test_codebuild_batch_get_builds_1_project():
+    client = boto3.client("codebuild", region_name="eu-central-1")
+
+    name = "some_project"
+    source = dict()
+    source["type"] = "S3"
+    # repository location for S3
+    source["location"] = "bucketname/path/file.zip"
+    artifacts = {"type": "NO_ARTIFACTS"}
+
+    environment = dict()
+    environment["type"] = "LINUX_CONTAINER"
+    environment["image"] = "contents_not_validated"
+    environment["computeType"] = "BUILD_GENERAL1_SMALL"
+    service_role = (
+        "arn:aws:iam::{0}:role/service-role/my-codebuild-service-role".format(
+            ACCOUNT_ID
+        )
+    )
+
+    client.create_project(
+        name=name,
+        source=source,
+        artifacts=artifacts,
+        environment=environment,
+        serviceRole=service_role,
+    )
+    client.start_build(projectName=name)
+
+    history = client.list_builds_for_project(projectName=name)
+    response = client.batch_get_builds(ids=history["ids"])
+
+    response.should_not.be.none
+    response["builds"].should_not.be.none
+    response["builds"][0]["currentPhase"].should.equal("COMPLETED")
+    response["builds"][0]["buildNumber"].should.be.a(int)
+    response["builds"][0]["phases"].should_not.be.none
+    len(response["builds"][0]["phases"]).should.equal(11)
+
+
+@mock_codebuild
+def test_codebuild_batch_get_builds_2_projects():
+    client = boto3.client("codebuild", region_name="eu-central-1")
+
+    source = dict()
+    source["type"] = "S3"
+    # repository location for S3
+    source["location"] = "bucketname/path/file.zip"
+    artifacts = {"type": "NO_ARTIFACTS"}
+
+    environment = dict()
+    environment["type"] = "LINUX_CONTAINER"
+    environment["image"] = "contents_not_validated"
+    environment["computeType"] = "BUILD_GENERAL1_SMALL"
+    service_role = (
+        "arn:aws:iam::{0}:role/service-role/my-codebuild-service-role".format(
+            ACCOUNT_ID
+        )
+    )
+
+    client.create_project(
+        name="project-1",
+        source=source,
+        artifacts=artifacts,
+        environment=environment,
+        serviceRole=service_role,
+    )
+    client.start_build(projectName="project-1")
+
+    client.create_project(
+        name="project-2",
+        source=source,
+        artifacts=artifacts,
+        environment=environment,
+        serviceRole=service_role,
+    )
+    client.start_build(projectName="project-2")
+
+    response = client.list_builds()
+    response["ids"].should_not.be.empty
+
+    for build_id in response["ids"]:
+        try:
```

Comment:
Similar comment here - to keep the same style, let's use `build_id.should.be.within([..])`